compressed_image_transport/cfg/CompressedSubscriber.cfg

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python
+
+PACKAGE='compressed_image_transport'
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+mode_enum = gen.enum( [gen.const("unchanged", str_t, "unchanged", "keep image encoding"),
```

Comment:
though that code is valid, I'd rather see int_t for the types and 0,1,2 for values: that will prevent runtime errors because strings can have typos